packages/storage-events/src/replay.js

```diff
@@ -1,16 +1,17 @@
+const { moveTodo } = require("./move-todo.js");
+
+module.exports = replay;
+
 function replay(events) {
   const todos = events.reduce((todos, event) => {
     switch (event.eventType) {
       case "ADDED_TODO":
         return addTodo(todos, event.data);
       case "REMOVED_TODO":
         return removeTodo(todos, event.data);
-      case "CHANGED_TODO":
-        if (event.data.parentId !== undefined) {
-          const result = moveTodo(todos, event);
-          if (result.moved) {
-            return result.result;
-          }
+      case "CHANGED_TODO": // we should probably make this MOVE_TODO and CHANGE_TODO
```

Comment:
Would be okay for me, but then it should be done with this change here as well as it will also be breaking.

Reply:
Ok, will add that change